### ISSUE TITLE
make DOI recommendation more abstract

### ISSUE THREAD

**Issue by tomkralidis** [wmo-im/wcmp2#38]
Informal feedback from KNMI (2022-08-08):

> 7.1.12 Digital Object Identifier => I understand what is meant, but it may be better to avoid the “DOI” in this case, as you want to leave room for other persistent identifiers, such as Handle of ARK. So I would suggest to call this “ Persistent Identifier”  or Resolvable Persistent Identifier. And then mention DOI as one of the examples.

**Comment by tomkralidis** [comment]
Given our model already supports this approach, it would make sense to make the text more abstract and provide DOI, Handle, ARK, etc. examples, i.e.:

```json
"externalIds": [
  {
    "scheme": "doi",
    "value": "https://dx.doi.org/10.14287/10000001"
  },
  {
    "scheme": "hdl",
    "value": "2381/12775"
  },
  {
    "scheme": "ark",
    "value": "ark:/13030/tf5p30086k"
  }
]
```

**Comment by steingod** [comment]
I support this, but would also recommend identification of purpose using a controlled terminology that is also referenced (including with identifiers) so semantic mappings is possible. Is the externalId referring to a dataset landing page, a site description in Oscar, a product manual (e.g. for remote sensing products) a scientific paper etc. Else you will just a long list of Ids that you don't know the purpose of and cannot use in decision support systems.

**Comment by tomkralidis** [comment]
@steingod would the scheme support the identification of purpose (assuming a controlled vocabulary of ID schemes)?

**Comment by steingod** [comment]
Well I see now that my use case here is more appropriate to further overload the links element. As the documentation of this element strictly refers to "Persistent identifiers or handles for the dataset" I think the approach indicated above is sufficient.

**Comment by tomkralidis** [comment]
Addressed in #70
